components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -93,8 +93,48 @@ class TabsUseCases(
         }
     }
 
+    class RemoveAllTabsUseCase internal constructor(
```

Comment:
If you could write a few basic tests for these use cases as well, that would be great too. (: 

We have some handy ones in [`TabsUseCasesTest.kt`](https://github.com/mozilla-mobile/android-components/blob/master/components/feature/tabs/src/test/java/mozilla/components/feature/tabs/TabsUseCasesTest.kt) for reference.

Reply:
sure